network-api/networkapi/tests.py

```diff
@@ -216,3 +224,242 @@ def test_sends_x_robots_tag(self):
         xrobotstag_middleware = XRobotsTagMiddleware(MagicMock())
         response = xrobotstag_middleware(MagicMock())
         response.__setitem__.assert_called_with('X-Robots-Tag', 'noindex')
+
+
+class TestPNIAirtableConnections(TestCase):
+    """
+    This set of tests is used purely to make sure the fieldnames and mappings for PNI products to connect
+    to Airtable don't change. No other tests are being performed here, it's simply looking at the import mapping
+    and the export field dictionaries.
+
+    If any of these fail, we know something in our code has changed and needs to be reflected in Airtable.
+    """
+
```

Comment:
These tests are to make sure we get some sort of automated flag if a field name changes in Wagtail.